src/contracts/Controller/AccessCheckController.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Contracts\User\Controller;
+
+interface AccessCheckController
```

Comment:
`Interface` suffix is missing.Also please consider the following alternative names:


```suggestion
interface AccessLimitedControllerInterface
```

```suggestion
interface RestrictedControllerInterface
```